docs/config.json

```diff
@@ -1376,6 +1386,11 @@
           "title": "Authorization",
           "slug": "/architecture/authorization/"
         },
+        {
+          "title": "Automatic Agent Update",
```

Comment:
This name seems out of place with the rest of the theme.  I would name Update Management and move it lower in the list.

Reply:
renamed to `Agent Update Management`